packages/scorer/src/provider/model-graded/faithfulness.ts

```diff
@@ -0,0 +1,156 @@
+import { ScoringFn } from "../../interface/scorer";
```

Comment:
**Suggestion:** The documentation for the `llm-faithfulness` scorer should be improved to provide more details on the expected input/output structure, the prompt format, and the scoring mechanism. This will help users understand how to use the scorer and interpret the results. [Documentation]
```suggestion
/**
 * The `llm-faithfulness` scorer evaluates the faithfulness of a language model's output to the given context and claims.
 *
 * Input:
 * - `sample`: An object containing the sample data, including `inputs` and `expected` properties.
 * - `config`: An object containing the scorer configuration, including:
 *   - `type`: The type of scorer, which must be `"llm-faithfulness"`.
 *   - `name`: (optional) The name of the scorer.
 *   - `context`: The context information that the language model should use to evaluate the claims.
 *   - `claims`: An array of claims that the language model should evaluate against the context.
 *
 * Output:
 * - An array containing a single object with the following properties:
 *   - `score`: A number between 0 and 1 representing the faithfulness of the language model's output.
 *   - `name`: The name of the scorer.
 *   - `message`: A string containing the reasons for any claims that were not supported by the context.
 *
 * The scorer uses a prompt-based approach to evaluate the faithfulness of the language model's output. It sends a prompt to the language model that includes the context and the claims, and then processes the language model's response to determine the faithfulness score and the reasons for any unsupported claims.
 */
```